src/app/modules/main/chat_section/module.nim

```diff
@@ -764,7 +764,6 @@ proc addNewChat(
     sharedUrlsService,
   )
 
-  self.chatContentModules[chatDto.id].load(chatItem)
```

Comment:
The other part of the lazy loading. We don't load when we create the item anymore